lib/triBoxOverlap.js

```diff
@@ -0,0 +1,212 @@
+'use strict';
+
+/**
+ * Function and helpers for computing the intersection between a static triangle and a static AABB.
+ * Adapted from Tomas Akenine-Möller's public domain implementation: http://www.cs.lth.se/home/Tomas_Akenine_Moller/code/
```

Comment:
Still add this to LICENSE.md as public domain so all third-party code is referenced from there.